hw2/01-coins.js

```diff
@@ -11,5 +37,13 @@ console.log(calculateChange(9.74));
 // $9.74 ==> 9 dollars, 2 quarters, 2 dimes, 4 pennies
```

Comment:
Hey @Saylee119, I ran your code and cross checked it with the expected output and it mismatches. You must ignore adding the change if its count is 0. Like for the above example 
Expected output: `9 dollars, 2 quarters, 2 dimes, 4 pennies`
Your output: `9 dollars, 2 quarters, 2 dime, 0 nikels, 4 pennies`. 
Can you handle such cases?